[Bug Report]: Start App Trace fails for esp32s2 on Windows - 1) cannot create trace file 2) invalid openocd command (VSC-525)

**Describe the bug**
I can run openocd from vs code on windows 10 with esp32s2 module and read App Trace using telnet 4444 port.
When pressing Start App Trace from vs code 2 errors are reported
1) it cannot create trace file - folder exists, permissions are ok
2) it issues invalid openocd command name "esp32", which is wrong in the esp32s2 context

Projects target is set esp32s2, and even workspace setting "idf.adapterTargetName" is set to "esp32s2".
Bellow is the output.

`Open On-Chip Debugger  v0.10.0-esp32-20200709 (2020-07-09-08:54)
Licensed under GNU GPL v2
For bug reports, read
	http://openocd.org/doc/doxygen/bugs.html
adapter speed: 20000 kHz

Info : Listening on port 6666 for tcl connections
Info : Listening on port 4444 for telnet connections
Info : ftdi: if you experience problems at higher adapter clocks, try the command "ftdi_tdo_sample_edge falling"
Info : clock speed 20000 kHz
Info : JTAG tap: esp32s2.cpu tap/device found: 0x120034e5 (mfg: 0x272 (Tensilica), part: 0x2003, ver: 0x1)
Info : Listening on port 3333 for gdb connections
Info : accepting 'tcl' connection on tcp/6666
Info : dropped 'tcl' connection
Info : accepting 'tcl' connection on tcp/6666
Info : Total trace memory: 16384 bytes
Info : Open file /c%3A/g/e3/projects/esp32_catm_rc1180/trace/trace_1609778610494.trace
❌ Error: Failed to ❌ open file /c%3A/g/e3/projects/esp32_catm_rc1180/trace/trace_1609778610494.trace
Error: apptrace: Failed to init trace data destination 'file:///c%3A/g/e3/projects/esp32_catm_rc1180/trace/trace_1609778610494.trace'!
Error: Invalid 'cmd_❌ data->poll_period' arg!
Error: Not enough args! Need 1 trace data destinations!
Error: Failed to init cmd ctx (-4)!
Total trace memory: 16384 bytes
Open file /c%3A/g/e3/projects/esp32_catm_rc1180/trace/trace_1609778610494.trace
Failed to open file /c%3A/g/e3/projects/esp32_catm_rc1180/trace/trace_1609778610494.trace
apptrace: Failed to init trace data destination 'file:///c%3A/g/e3/projects/esp32_catm_rc1180/trace/trace_1609778610494.trace'!
Invalid 'cmd_data->poll_period' arg!
Not enough args! Need 1 trace data destinations!
Failed to init cmd ctx (-4)!
Info : accepting 'tcl' connection on tcp/6666
command.c:795: Error: invalid command name "esp32"
at file "command.c", line 795
❌ command.c:795: Error: invalid command name "esp32"
at file "command.c", line 795`

**To Reproduce**
Create esp-idf project in vs code, and some trace statements in code e.g. 
esp_apptrace_write(ESP_APPTRACE_DEST_TRAX, "Hello", 5, ESP_APPTRACE_TMO_INFINITE);
esp_apptrace_flush(ESP_APPTRACE_DEST_TRAX, ESP_APPTRACE_TMO_INFINITE);
set target to esp32s2
connect esp32s2 module to esp-prog programming and jtag connectors
build, flash and monitor
Start OpenOCD server from vs code
Press Start App Trace

**Expected behavior**
Expect to view the Hello message in the output

- OS Name & Version: Windows 10
- VSCode Version: 1.52.1
- ESP-IDF Version: 4.2
- Python Version: 3.8
- ESP-IDF vs code extention: 0.60

**Logs**
If applicable, please share the log file which can be obtained from
- Windows: `%USERPROFILE%\.vscode\extensions\espressif.esp-idf-extension-VERSION\esp_idf_vsc_ext.log`
{"message":"Open On-Chip Debugger  v0.10.0-esp32-20200709 (2020-07-09-08:54)\r\nLicensed under GNU GPL v2\r\nFor bug reports, read\r\n\thttp://openocd.org/doc/doxygen/bugs.html\r\n","stack":"Error: Open On-Chip Debugger  v0.10.0-esp32-20200709 (2020-07-09-08:54)\r\nLicensed under GNU GPL v2\r\nFor bug reports, read\r\n\thttp://openocd.org/doc/doxygen/bugs.html\r\n\n\tat c:\\Users\\User\\.vscode\\extensions\\espressif.esp-idf-extension-0.6.0\\dist\\extension.js:1:7720\n\tat ChildProcess.exithandler (child_process.js:295:7)\n\tat ChildProcess.emit (events.js:223:5)\n\tat maybeClose (internal/child_process.js:1021:16)\n\tat Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)","level":"error","timestamp":"2021-01-04T16:57:54.055Z"}

--- a/src/espIdf/openOcd/boardConfiguration.ts
+++ b/src/espIdf/openOcd/boardConfiguration.ts
@@ -32,13 +32,13 @@ const defaultBoards = [
     name: "ESP32 module",
     description: "ESP32 used with ESP-PROG board",
     target: "esp32",
-    configFiles: ["interface/ftdi/esp32_devkitjv1.cfg", "target/esp32.cfg"],
+    configFiles: ["interface/ftdi/esp32_devkitj_v1.cfg", "target/esp32.cfg"],
   } as IdfBoard,
   {
     name: "ESP32-S2 module",
     description: "ESP32-S2 used with ESP-PROG board",
     target: "esp32s2",
-    configFiles: ["interface/ftdi/esp32_devkitjv1.cfg", "target/esp32s2.cfg"],
+    configFiles: ["interface/ftdi/esp32_devkitj_v1.cfg", "target/esp32s2.cfg"],
   } as IdfBoard,
 ];
 
@@ -76,7 +76,7 @@ export async function getBoards() {
       name: "Custom board",
       description: "No board selected",
       target: "esp32",
-      configFiles: ["interface/ftdi/esp32_devkitjv1.cfg", "target/esp32.cfg"],
+      configFiles: ["interface/ftdi/esp32_devkitj_v1.cfg", "target/esp32.cfg"],
     } as IdfBoard;
     espBoards.push(emptyBoard);
     return espBoards;

--- a/src/espIdf/tracing/appTraceManager.ts
+++ b/src/espIdf/tracing/appTraceManager.ts
@@ -191,7 +191,7 @@ export class AppTraceManager extends EventEmitter {
   public async stop() {
     if (await OpenOCDManager.init().promptUserToLaunchOpenOCDServer()) {
       this.shallContinueCheckingStatus = false;
-      const stopHandler = this.sendCommandToTCLSession("esp32 apptrace stop");
+      const stopHandler = this.sendCommandToTCLSession("esp apptrace stop");
       stopHandler.on("response", (resp: Buffer) => {
         const respStr = resp.toString();
         if (respStr.includes("Tracing is not running!")) {
@@ -268,7 +268,7 @@ export class AppTraceManager extends EventEmitter {
     });
     const statusCheckerTimer = setInterval(() => {
       if (this.shallContinueCheckingStatus) {
-        tclClient.sendCommandWithCapture("esp32 apptrace status");
+        tclClient.sendCommandWithCapture("esp apptrace status");
       } else {
         clearInterval(statusCheckerTimer);
       }

--- a/src/extension.ts
+++ b/src/extension.ts
@@ -1203,7 +1203,7 @@ export async function activate(context: vscode.ExtensionContext) {
       const boards = await getBoards();
       const choices = boards.map((b) => {
         return {
-          description: `${b.description} (${b.configFiles.join(",")})`,
+          description: `${b.description} (${b.configFiles})`,
           label: b.name,
           target: b,
         };
@@ -1229,7 +1229,7 @@ export async function activate(context: vscode.ExtensionContext) {
       }
       await idfConf.writeParameter(
         "idf.openOcdConfigs",
-        selectedBoard.target.configFiles.join(","),
+        selectedBoard.target.configFiles,
         target
       );
       Logger.infoNotify("OpenOCD Board configuration files are updated.");

